src/audio/base_fw.c

```diff
@@ -467,6 +467,45 @@ static int global_perf_data_get(uint32_t *data_off_size, char *data)
 #endif
 }
 
+static int io_global_perf_state_get(uint32_t *data_off_size, char *data)
```

Comment:
`size_t`? `char` is also a bit confusing for `data`, why not use the appropriate `enum`